README.md

```diff
@@ -91,7 +91,7 @@ $ cd rmm
 - Create the conda development environment `rmm_dev`
 ```bash
 # create the conda environment (assuming in base `rmm` directory)
-$ conda env create --name rmm_dev --file conda/environments/rmm_dev_cuda11.0.yml
+$ conda env create --name rmm_dev --file conda/environments/all_cuda-115_arch-$(arch).yaml
```

Comment:
We only generate files for `x86_64` so we should hard-code that. ARM environments don't exist.
```suggestion
$ conda env create --name rmm_dev --file conda/environments/all_cuda-115_arch-x86_64.yaml
```